.storybook/manager-head.html

```diff
@@ -44,24 +47,30 @@
   //workaround to increase the left panel width just enough to show even the longer story names
```

Comment:
just want to make sure - this is only eslint changes?

Reply:
Yes, I changes only content inside <style> tag